platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/migratable/BundleManagerTest.java

```diff
@@ -0,0 +1,217 @@
+
+  private final BundleContext context = Mockito.mock(BundleContext.class);
+
+  private final ProfileMigrationReport report = Mockito.mock(ProfileMigrationReport.class);
+
+  @Before
+  public void setup() throws Exception {
+    Mockito.doReturn(context).when(bundleMgr).getBundleContext();
+
+    Mockito.when(bundle.getBundleId()).thenReturn(ID);
+    Mockito.when(bundle.getSymbolicName()).thenReturn(NAME);
+    Mockito.when(bundle.getVersion()).thenReturn(VERSION);
+    Mockito.when(bundle.getState()).thenReturn(STATE);
+    Mockito.when(bundle.getLocation()).thenReturn(LOCATION);
+
+    Mockito.when(bundle2.getBundleId()).thenReturn(ID2);
+    Mockito.when(bundle2.getSymbolicName()).thenReturn(NAME2);
+    Mockito.when(bundle2.getVersion()).thenReturn(VERSION2);
+    Mockito.when(bundle2.getState()).thenReturn(STATE2);
+    Mockito.when(bundle2.getLocation()).thenReturn(LOCATION2);
+
+    Mockito.when(bundle3.getBundleId()).thenReturn(ID3);
+    Mockito.when(bundle3.getSymbolicName()).thenReturn(NAME3);
+    Mockito.when(bundle3.getVersion()).thenReturn(VERSION3);
+    Mockito.when(bundle3.getState()).thenReturn(STATE3);
+    Mockito.when(bundle3.getLocation()).thenReturn(LOCATION3);
+
+    Mockito.when(bundle4.getBundleId()).thenReturn(ID4);
+    Mockito.when(bundle4.getSymbolicName()).thenReturn(NAME4);
+    Mockito.when(bundle4.getVersion()).thenReturn(VERSION4);
+    Mockito.when(bundle4.getState()).thenReturn(STATE4);
+    Mockito.when(bundle4.getLocation()).thenReturn(LOCATION4);
+
+    Mockito.when(bundleMgr.newTaskList(report)).thenReturn(tasks);
+  }
+
+  @Test
+  public void testExportBundles() throws Exception {
+    Mockito.when(bundleProcessor.listBundles(context))
+        .thenReturn(new Bundle[] {bundle, bundle2, bundle3});
+
+    Assert.assertThat(
+        bundleMgr.exportBundles(),
+        Matchers.contains(
+            Matchers.allOf( //
+                Matchers.aMapWithSize(5),
+                Matchers.hasEntry(ProfileMigratable.PROFILE_ID, (Object) ID),
+                Matchers.hasEntry(ProfileMigratable.PROFILE_NAME, NAME),
+                Matchers.hasEntry(
+                    Matchers.equalTo(ProfileMigratable.PROFILE_VERSION),
+                    (Matcher<Object>)
+                        (Matcher)
+                            Matchers.contains(
+                                VERSION.getMajor(),
+                                VERSION.getMinor(),
+                                VERSION.getMicro(),
+                                VERSION.getQualifier())),
+                Matchers.hasEntry(ProfileMigratable.PROFILE_STATE, (Object) STATE),
+                Matchers.hasEntry(ProfileMigratable.PROFILE_LOCATION, LOCATION)),
+            Matchers.allOf( //
+                Matchers.aMapWithSize(5),
+                Matchers.hasEntry(ProfileMigratable.PROFILE_ID, (Object) ID2),
+                Matchers.hasEntry(ProfileMigratable.PROFILE_NAME, NAME2),
+                Matchers.hasEntry(
```

Comment:
✏️ These test classes would be more readable if the static methods were imported instead.

Reply:
Yeah! Eric and Steve have been trying to get me to do that 😆  I just cannot get myself to do that.